pkg/parsers/parsers.go

```diff
@@ -12,17 +12,29 @@ import (
 	"strings"
 )
 
+var (
+	// DefaultHTTPHost Default HTTP Host used if only port is provided to -H flag e.g. docker -d -H tcp://:8080
+	DefaultHTTPHost = "127.0.0.1"
```

Comment:
How about move ParseHost() and ParseTCPAddr(), ParseUnixAddr() it relies on to opt, cause its only used in opt?  @runcom